FAQ.md

```diff
@@ -209,6 +227,39 @@ If we wanted to only warn for things that throw, then we would warn when calling
  
 NOTE(domenic): there is a conflict between static analysis and dynamic analysis. Consider the example let x : number = true ? 0 : false; A runtime type system would have no problem with this, whereas what you're proposing would disallow it. This is probably worth highlighting at a higher level in the document: it seems like you are leaning toward a statically-checkable model, which there is precedent for for JS but I believe other optionally-typed languages went a different direction.
 
+## Are you breaking the web?
+
+Nope. See [are all existing programs correct?](#are-all-existing-programs-correct).
```

Comment:
I would also add some text about what it means to be a conforming JS implementation: parsing the new syntax, giving early errors in the new syntax, and nowhere else. A separate tool, the typechecker, is responsible for checking the type semantics we are proposing.